plugins/tiddlywiki/dynannotate/examples/viewtemplate.tid

```diff
@@ -1,28 +1,72 @@
-tags: $:/tags/dynannotateExamples
 caption: View Template
+tags: $:/tags/dynannotateExamples
+title: $:/plugins/tiddlywiki/dynannotate/examples/viewtemplate
+
+\define show-example(example)
+<$codeblock code=<<__example__>>/>
+
+//''Displays as:''//
+
+$example$
+\end
 
 <div class="tc-dynannotation-example-info">
 
 !! Using Dynannotate in the view template
 
-This example shows how to override the core view template with a custom template that includes dynannotate. It is disabled by default but can be enabled by clicking the button below. (Clicking the button below copies the custom view template from $:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text to $:/core/ui/ViewTemplate/body).
+This example shows how to override the core view template with a custom template that includes dynannotate. It is disabled by default but can be enabled by clicking the corresponding button below. The example transcludes the [[Searching in TiddlyWiki|$:/plugins/tiddlywiki/dynannotate/searching-in-tiddlywiki]] tiddler for illustration purposes, but the chosen view template is applied to all open tiddlers.
+
+Once enabled, this example demonstrates several features of Dynannotate:
 
-Once enabled, this example demonstrates several features:
+* Highlight text within tiddler bodies -- any text entered in the sidebar search input will be highlighted (in all tiddlers)
+* Create annotations (on any tiddler) by selecting text and then clicking //Create annotation// in the resulting dropdown
 
-* Highlighting of search results within tiddler bodies
-* Creating annotations on any tiddler by selecting text and then clicking a colour in the resulting dropdown
+(Clicking the buttons below either puts the custom body view template [[$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text]] in front of the default in the [[View Template Body Cascade|https://tiddlywiki.com/#View%20Template%20Body%20Cascade]] or removes it from the cascade).
 
 </div>
 
 <$button>
-<$action-setfield $tiddler="$:/config/ViewTemplateTitleFilters/dynannotate" tags="$:/tags/ViewTemplateTitleFilter" text="[[$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text]]" list-before="$:/config/ViewTemplateBodyFilters/default"/>
+<$action-setfield $tiddler="$:/config/ViewTemplateBodyFilters/dynannotate" tags="$:/tags/ViewTemplateBodyFilter" text="[[$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text]]" list-before="$:/config/ViewTemplateBodyFilters/default"/>
 Click here to use the custom view template
 </$button>
 
 <$button>
-<$action-deletetiddler $tiddler="$:/config/ViewTemplateTitleFilters/dynannotate"/>
+<$action-deletetiddler $tiddler="$:/config/ViewTemplateBodyFilters/dynannotate"/>
 Click here to revert to the core view template
 </$button>
 
-<$codeblock code={{$:/plugins/tiddlywiki/dynannotate/examples/viewtemplate/text}}/>
\ No newline at end of file
+<div style="border: 1px solid black; border-radius: 20px; padding: 1em;">
```

Comment:
Could we use a class here?

Reply:
I was afraid you'd say that. There's just no reward for being lazy... 😉 
Done and done.